<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_restaurant_div_pop</class>
 <widget class="QDialog" name="w_scr_dish_restaurant_div_pop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_ch_tableno">
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ftableview" name="tableView"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>目前单号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_desc">
          <property name="text">
           <string/>
          </property>
          <property name="outer_stylesheet" stdset="0">
           <string>label_blue</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>9</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="lds_toolbar_button" name="pushButton_ok">
          <property name="text">
           <string>确定</string>
          </property>
          <property name="outer_stylesheet" stdset="0">
           <string>pushbutton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="lds_toolbar_button" name="pushButton_cancel">
          <property name="text">
           <string>退出</string>
          </property>
          <property name="outer_stylesheet" stdset="0">
           <string>pushbutton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ftableview</class>
   <extends>QTableView</extends>
   <header>ftableview.h</header>
  </customwidget>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
